yarn-project/aztec/src/cli/cmds/start_prover_agent.ts

```diff
@@ -61,6 +64,16 @@ export async function startProverAgent(
       ),
   );
 
+  // expose all agents as individual services
+  for (let i = 0; i < agents.length; i++) {
+    services[`agent${i}`] = [agents[i], ProverAgentApiSchema, () => agents[i].getStatus().status !== 'stopped'];
+  }
+
+  // shortcut in the most common case of having a single running agent
+  if (agents.length === 1) {
+    services[`agent`] = [agents[0], ProverAgentApiSchema, () => agents[0].getStatus().status !== 'stopped'];
+  }
```

Comment:
We need to have a service exposed through the `services` object in order to get an http server running. Without an http server running on the RPC port the agent pod would fail health checks